docs/src/main/sphinx/connector/hive.rst

```diff
@@ -380,6 +380,10 @@ Property Name                                      Description
 ``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views. You can    ``false``
                                                    alternatively set the ``legacy_hive_view_translation``
                                                    session property to ``true``.
+
+``hive.parallel-partitioned-bucketed-inserts``     Improve parallelism of partitioned and bucketed table        ``true``
+                                                   inserts. When disabled, the number of writers used
+                                                   will be limited to number of buckets.
```

Comment:
buckets or partitions?

also "number of writers" or of "writing threads"?

please document whether `hive.parallel-partitioned-bucketed-inserts` affects the number of files being created.

Reply:
when disabled it's limited to number of buckets. Number of files does not change, so I'm not sure anything needs to be documented.

Reply:
> when disabled it's limited to number of buckets.

and when enabled ?

Reply:
When enabled. it's limited to number of partitions * number of buckets or number of total writers in the cluster. Do you think I should extend the note somehow? I could do that in separate PR, since it's tricky to describe.